From 3bcefae1376d29dc10881b55701cf0aab71ce380 Mon Sep 17 00:00:00 2001
From: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
Date: Thu, 18 Apr 2019 09:14:49 +0200
Subject: [PATCH 2/8] osm_ucast_ftree.c: Fix clang warning about empty loop

osm_ucast_ftree.c:3861:52: warning: for loop has empty body [-Wempty-body]
                     && (!p_hca->up_port_groups[i]->is_cn); i++) ;
                                                                 ^
osm_ucast_ftree.c:3861:52: note: put the semicolon on a separate line to silence this warning

Signed-off-by: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
Signed-off-by: Hal Rosenstock <hal@mellanox.com>
---
 opensm/osm_ucast_ftree.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/opensm/osm_ucast_ftree.c b/opensm/osm_ucast_ftree.c
index 7fa78efa..106ef6bf 100644
--- a/opensm/osm_ucast_ftree.c
+++ b/opensm/osm_ucast_ftree.c
@@ -3866,7 +3866,9 @@ static void fabric_set_leaf_rank(IN ftree_fabric_t * p_ftree)
 			hca_get_guid_ho(p_hca));
 
 		for (i = 0; (i < p_hca->up_port_groups_num)
-		     && (!p_hca->up_port_groups[i]->is_cn); i++) ;
+		     && (!p_hca->up_port_groups[i]->is_cn); i++)
+			;
+
 		CL_ASSERT(i < p_hca->up_port_groups_num);
 		CL_ASSERT(p_hca->up_port_groups[i]->remote_node_type ==
 			  IB_NODE_TYPE_SWITCH);
-- 
2.20.1

